software/base/src/test/java/org/apache/brooklyn/entity/brooklynnode/BrooklynNodeIntegrationTest.java

```diff
@@ -578,7 +578,7 @@ public void testUsesLocation() throws Exception {
         assertEquals(parseJson(locatedLocationsContent, ImmutableList.of(appLocationId, "longitude"), Double.class), 45.6, 0.00001);
     }
 
-    @Test(groups="Integration")
+    @Test(groups={"Integration", "Broken"})
```

Comment:
What is causing all of these tests to fail? Any clues that could be given in a short comment?
I just tried running this one, and it passed locally for me (on OSX).